### DISCUSSION TITLE
Testing RS485 comms

### DISCUSSION THREAD

**Discussion by kenironside** [kellerza/sunsynk#152] [Q&A]
Hi, newbie question, how can I test RS485 comms between my inverter and my HA device? I have a RS485 to USB dongle between my Sunsynk inverter and RasPi HA install. The HA install works and I have installed the Energy dashboard but I'm not sure I'm getting live data. I also have the Sunsynk data from cloud integration & Power Card dashboard and I'm not clear how to see what data is going to what integration. 
Thanks for any assistance. regards, Ken

**Comment by kellerza** [comment]
The Energy dashboard values are not live. They are delayed (summarized) by at least an hour, so maybe this is already solved.

If you look into the Power Card (see the website for config examples), these will be live
